nestjs/src/courses/cross-checks/course-cross-checks.service.ts

```diff
@@ -186,7 +187,10 @@ export class CourseCrossCheckService {
       solutionUrl: data.ts_url,
     }));
 
-    return result;
+    // Each record occurs n times where n is number of checkers
+    const deduplicatedRecords = uniqBy(result, 'githubId');
+
+    return deduplicatedRecords;
```

Comment:
if we take data immediately from the task_solution table, it seems to me that we will not need to remove duplicates here

Reply:
Good catch, gonna try!